modules/gitops-disable-dex.adoc

```diff
@@ -1,22 +1,28 @@
 // Module is included in the following assemblies:
 //
-// * configuring-sso-for-argo-cd-on-openshift
+// * configuring-sso-for-argo-cd-using-dex
 
 :_content-type: PROCEDURE
 [id="gitops-disable-dex_{context}"]
 = Disabling Dex
 
-Dex is installed by default for all the Argo CD instances created by the Operator. You can disable Dex.
+Dex is installed by default for all the Argo CD instances created by the Operator. You can configure {gitops-title} to use Dex as the SSO authentication provider by setting the `.spec.dex` parameter. 
+
+[IMPORTANT]
+====
+In {gitops-title} v1.6.0, `DISABLE_DEX` is deprecated and is planned to be removed in {gitops-title} v1.9.0. Consider using the `.spec.sso.dex` parameter instead. See "Enabling or disabling Dex using .spec.sso".
+====
 
 .Procedure
 
-* Set the environmental variable `DISABLE_DEX` to true in the *YAML* resource of the Operator:
+* Set the environmental variable `DISABLE_DEX` to true in the YAML resource of the Operator:
 +
 [source,yaml]
 ----
-  spec:
+spec:
   config:
     env:
     - name: DISABLE_DEX
-      value: "true"
+      value: "false"
```

Comment:
Line 18 and 26 are at odds with each other (true vs false). Also the logic is a little twisted. If you set "disable dex" to "true," it seems like that should disable Dex, right?